app/graphql/mutations/profile/associate_systems.rb

```diff
@@ -13,9 +13,10 @@ class AssociateSystems < BaseMutation
 
       def resolve(args = {})
         ::Profile.transaction do
+          hosts = find_hosts(args[:system_ids])
           profile = find_profile(args[:id])
           if profile&.policy
-            profile.policy.update_hosts(args[:system_ids])
+            profile.policy.update_hosts(hosts.pluck(:id))
```

Comment:
Question about this change.. Is this more efficient than the way it was before? Just curious